syntax = "proto3";

package audio;

service AudioService {
  rpc ProcessAudio (AudioRequest) returns (AudioResponse);
}

message AudioRequest {
  bytes audio = 1;
}

message AudioResponse {
  bytes audio = 1;
}